Editor/UI/ProjectAuditorWindow.cs

```diff
@@ -1296,7 +1290,22 @@ void DrawToolbar()
 #endif
                 Utility.ToolbarDropdownList(m_ViewDropdownItems,
                     m_ViewManager.activeViewIndex,
-                    (category) => {m_ViewManager.ChangeView((IssueCategory)category);}, GUILayout.Width(largeButtonWidth));
+                    (arg) =>
+                    {
+                        var category = (IssueCategory)arg;
+                        if (m_ProjectReport == null)
+                            return; // this happens from the summary view while the report is being generated
+                        if (!m_ProjectReport.HasCategory(category))
+                        {
+                            var displayName = m_ViewManager.GetView(category).desc.displayName;
+                            if (!EditorUtility.DisplayDialog(k_ProjectAuditorName, $"'{displayName}' analysis will now begin", "Ok", "Cancel"))
```

Comment:
Add a point at the end of the sentence?
```suggestion
                            if (!EditorUtility.DisplayDialog(k_ProjectAuditorName, $"'{displayName}' analysis will now begin.", "Ok", "Cancel"))
```